src/Engine/EngineBase.php

```diff
@@ -154,10 +199,19 @@ public function getLangName( ?string $lang = null ): string {
 			: $this->intuition->getLangName( $lang );
 	}
 
+	/**
+	 * Get name of the given line detection model
+	 * @param string|null $lineIdLang
+	 * @return string
+	 */
+	public function getLineIdModelName( ?string $lineIdLang = null ): string {
+		return self::LINE_ID_MODEL_NAMES[$lineIdLang];
+	}
+
 	/**
 	 * Transform the given ISO 639-1 codes into the language codes needed by this type of Engine.
```

Comment:
This comment, and perhaps the method name as well, should probably be updated to reflect that the return value is 'language model information' or something, rather than just "the language codes".

Reply:
I'm not sure I get you on this. This method just returns the name of the model from the array `LINE_ID_MODEL_NAMES`.